### DISCUSSION TITLE
Does your Sheet (drawer) animation work?

### DISCUSSION THREAD

**Discussion by cwik-tech** [shadcn-ui/ui#1605] [Q&A]
I am trying to implement the Sheet component and it all works, but it is not animated at all. From the code I would presume it should be animated, but it just appears and disapears. I used the manual way of installing the component. Any suggestions?

**Comment by csalmeida** [comment]
In my case the animations did not work because the [Tailwind Animate](https://github.com/jamiebuilds/tailwindcss-animate) package was not installed.

I imported it into `tailwind.config.js`:

```js
import animate from 'tailwindcss-animate';
```

And added it to the plugins array:

```js
{
  // config code here 
  plugins: [forms, typography, animate],
  // other code here
}
```

**Reply by shantanu-scaura** [reply]
@csalmeida thank you! That worked for me.

In my case, the plugin was added as following:
```js
{
    plugins: [import('tailwindcss-animate')],
}
```

I do not remember making changes here so I assume it was this way by default - maybe it was updated in the CLI at some point.
Either ways, importing `animate` specifically from `'tailwindcss-animate'` did the trick for me. Thanks again!

```js
import animate from 'tailwindcss-animate'
```

```js
{
    plugins: [animate],
}
```

**Reply by csalmeida** [reply]
That's great, I'm glad it helped and thanks for sharing the steps you followed as well.
